src/test/resources/processor/chunker/TextChunkingTestMissingFieldDocument.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Can we remove this extra file and instead declare string and convert to json in the code itself?